app/views/cookbooks/directory.html.erb

```diff
@@ -80,6 +80,9 @@
       <code>knife cookbook site</code> commands. <%= link_to 'The documentation', chef_docs_url('knife_cookbook_site.html') %> for <code>knife cookbook site</code> explains how to
       download, share, unshare, search, install and list cookbooks on Supermarket.
       You can also interact with the <%= link_to 'Supermarket API', chef_docs_url('api_cookbooks_site.html') %> directly.</p>
+
+      <h3>How do I adopt a cookbook?</h3>
+      <p><%= link_to 'Click here', available_for_adoption_path %> to view the list of adoptable cookbooks.</p>
```

Comment:
I have a personal distain for using the words "click here" in links.  It might just be me.  It is definitely a nit being picked.

    <p>Select a cookbook from the <%= link_to 'list of adoptable cookbooks' %> and click the "Adopt me!" button.</p>